src/query/management/src/user/user_api.rs

```diff
@@ -27,6 +28,8 @@ pub trait UserApi: Sync + Send {
 
     async fn get_users(&self) -> Result<Vec<SeqV<UserInfo>>>;
 
+    async fn get_raw_users(&self) -> Result<ListKVReply>;
```

Comment:
add a comment about why this method is introduced?

Reply:
Ok